### ISSUE TITLE
[Logging v2] - etcd and Control Plane dedicated nodes are not getting fluent-bit aggregator pods assigned

### ISSUE THREAD

**Issue by izaac** [rancher/rancher#29028]
**What kind of request is this:**
bug

**Steps to reproduce (least amount of steps as possible):**

- Install `rancher-logging` App chart in an RKE cluster with the following configuration
  - 1 etcd node / 1 Control Plane / 3 Workers
- etcd logs are only stored in etcd node

**Result:**

**etcd node**
```
root@izaac-do-testlog1-etcd-1:~# ls /var/lib/rancher/rke/log
etcd_97dac881d1f2f500715b11024180bb1587d29c7be1a3e235a5d72e983596d2f4.log        kubelet_edbd812c3dc14d63a504f47f8d287e6234f1f89dbbbe7fa33c174a019f5ea38d.log
kube-proxy_ebac9a6be5620b6db99903c77fba16ff2fb9feb3be8d153bdc4742820390448d.log  nginx-proxy_519296f4b3c964dc8c45f6a8de9d48ba2ad9e66ba279373b8598d4e2f0c12c37.log
```

fluentbit aggregators are only added to worker nodes and logs from the etcd is not flowing to the backend

**Taints of etcd node**

```
k describe node izaac-do-testlog1-etcd-1  | grep -i taint
Taints:             node-role.kubernetes.io/etcd=true:NoExecute
```

**Other details that may be helpful:**

Worker nodes do not store etcd logs

```
root@izaac-do-testlog1-wrkr-1:~# ls /var/lib/rancher/rke/log
kube-proxy_d1931ce5c040197cfaffd20a4d56527cb6d7b9fdbf62e27ad7e8371e07e31bac.log  nginx-proxy_cb6285cd512dd2774c682f9daa3ce639994977f6ec55d856a33e90f1e2b5570f.log
kubelet_b2a1b36fea3726de97ce2d65e295f1d786e7d61304ce4330d8cf57d579a9d8f5.log
```

**Environment information**
- Rancher version: `master-head (09/19/2020)` _bd88c23_
- Installation option: HA k8s with provisioned DO RKE downstream cluster.

**Comment by izaac** [comment]
This seems to be the case too for a dedicated RKE Control Plane node.

**Comment by paynejacob** [comment]
https://github.com/rancher/dashboard/pull/1328
https://github.com/rancher/charts/pull/673

**Comment by cbron** [comment]
Moving back to 2.5 since PR for fix already exists. If that fails test we can reassess.

**Comment by izaac** [comment]
Rancher version: `master-head (09/22/2020)` _0ccce45_

- `RKE DO  v1.19.2-rancher1-1
- `RKE Federal v1.18.8-beta20+rke2`
-  `k3s v1.19.2+k3s1`

EKS will be checked after this is fixed #29085 but as part of UI testing.


Logs are now coming from dedicated `etcd` and `Control Plane` nodes.
